core/domain/src/commonMain/kotlin/team/aliens/dms/kmp/core/domain/usecase/notice/GetWhetherNewNoticesExistUseCase.kt

```
@@ -0,0 +1,4 @@
+package team.aliens.dms.kmp.core.domain.usecase.notice
+
+class GetWhetherNewNoticesExistUseCase {
+}
```

detekt (1.23.7)

[warning] The class or object GetWhetherNewNoticesExistUseCase is empty.

(detekt.empty-blocks.EmptyClassBlock)